app/dashboard/templates/bounty/details.html

```diff
@@ -511,7 +511,7 @@ <h3>{{ noscript.keywords }}</h3>
   {% include 'shared/current_profile.html' %}
 
   <script>
-    const is_bounties_network = {{ is_bounties_network|lower }};
+    const is_bounties_network = {% if is_bounties_network %} {{ is_bounties_network|lower }} {% else %} true {% endif %};
```

Comment:
You can use here {% firstof is_bounties_network|lower or 'true' %}

Reply:
OHHH NICEEEEEEEEEE